script/Ciel/Day 9/09_01_CIEL09_3_2.txt

```diff
@@ -216,22 +216,22 @@ Just listening to his supplications is enough to deeply pain a man's heart.
 [sha:0b8bf555b3b8888a6d4548febe8b98e7d1693ff2]{
 -- Page 360, Offset 23064.
 -- あるいは、ソレは本気で更生しようと誓ったかもしれない。
-He may be vowing in all seriousness to repent himself.
+He may be earnestly vowing to reform himself.
 }
 [sha:f1a1cdefef841a467082fbf7da7519699674a096]{
 -- Page 360, Offset 23065.
 -- この悪夢、目の前の女に生きたまま■■される悪夢から抜け出せるなら罪を償うと。
-Vowing to atone for his sins if he can break free from this nightmare, the nightmare of being ■■ed while alive by the woman in front of him.
+Vowing to atone for his sins if he can break free from this nightmare, the nightmare of being ■■ed alive by the woman in front of him.
```

Comment:
I want to be ■■ed alive by Noel too :roa:

Reply:
![image](https://user-images.githubusercontent.com/97076550/188528570-e1617123-1875-411d-95c5-d7a594975163.png)